### PR TITLE
feat: add maxStringLength error message

### PR DESCRIPTION
**Pull request by MaxPostema**
Point to think about: Should this also be added to other fields like the textInput with some other default max limit?

### CODE REVIEW

**Review by connoratrug — CHANGES_REQUESTED** [review]
please update molgenis-components as well

**Comment by MaxPostema** [comment]
> please update molgenis-components as well
I was planning to add it to both! ;)

File: apps/styleguide/src/forms/InputString.vue [hunk 1]
```diff
@@ -38,22 +43,38 @@
...
   components: {
     InputAppend,
-    FormGroup: () => import('./_formGroup'), //because it uses itself in nested form
-    IconAction
+    FormGroup: () => import("./_formGroup"), // needed because it uses itself in nested forms
+    IconAction,
+  },
+  computed: {
+    stringError() {
+      console.log(this.valueArray);
```
Comment (connoratrug) [inline apps/styleguide/src/forms/InputString.vue:65]: remove console .log

File: apps/molgenis-components/src/components/forms/InputString.vue [hunk 2]
```diff
@@ -18,7 +18,7 @@
         @input="$emit('input', $event.target.value)"
         type="text"
         class="form-control"
-        :class="{ 'is-invalid': errorMessage }"
+        :class="{ 'is-invalid': stringError }"
```
Comment (connoratrug) [inline apps/molgenis-components/src/components/forms/InputString.vue:21]: what happend to the errorMessage , maybe ````stringError || errorMEssage```` ? or  maybe i'm missing something

File: apps/styleguide/src/forms/InputString.vue (hunk 1 above)
Comment (MaxPostema) [reply apps/styleguide/src/forms/InputString.vue]: 👍

File: apps/molgenis-components/src/components/forms/InputString.vue [hunk 3]
```diff
@@ -69,7 +84,10 @@ span:hover .hoverIcon {
     <b>column</b>
     <InputString id="input-string4" :label.sync="column.label" v-model="column.value" :editMeta="true"
                  :description.sync="column.description"/>
-    text.<br/>
+    text.<br/><br/>
+    <InputString id="input-string5" v-model="value" :stringLength="4" label="maximum stringLength (4)"/>
```
Comment (connoratrug) [inline apps/molgenis-components/src/components/forms/InputString.vue:88]: you are reusing the same value in the different string examples is this by accident or design ?

File: apps/molgenis-components/src/components/forms/InputString.vue (hunk 2 above)
Comment (MaxPostema) [reply apps/molgenis-components/src/components/forms/InputString.vue:21]: stringError is a computed that gives the specific string error or the more global errorMessage. See line 52 and 54.

**Review by connoratrug — CHANGES_REQUESTED** [review]
work in styleguide and components showcase but the error is not shown in the tables app

**Review by connoratrug — APPROVED** [review]

**Pull request merged**